demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -335,3 +336,44 @@ def is_there_spaces_in_the_end_of_id(self):
                 return False
 
         return True
+
+    def validate_readme_exists(self, caller: str, is_modified: bool, is_added: bool, validate_all: bool):
```

Comment:
use `self.structure_validator.file_type` instead of passing the `caller` argument.

Reply:
Done